virt/virtual_machines/virt-vtpm-devices.adoc

```diff
@@ -0,0 +1,32 @@
+:_content-type: ASSEMBLY
+[id="virt-vtpm-devices_{context}"]
+= Virtual trusted platform module devices
+include::_attributes/common-attributes.adoc[]
+:context: virt-vtpm-devices
+
+toc::[]
+
+:FeatureName: Virtual trusted platform module devices
+
+A Virtual Trusted Platform Module (vTPM) device protects virtual machines
+(VMs) by storing secrets similarly to a physical Trusted Platform Module (TPM)
+hardware chip.
```

Comment:
I think we could really use more conceptual information in this assembly; maybe a concept module? I searched the repo and found `modules/nbde-tpm-encryption.adoc` in the core docs, but I'm not sure if that information applies here. I'd suggest checking with an SME

Reply:
Do you mean more information about what a TPM is/does? Would it be considered too lazy to link the Wikipedia page for it? :) https://en.wikipedia.org/wiki/Trusted_Platform_Module